activesupport/lib/active_support/testing/parallelize_executor.rb

```diff
@@ -35,6 +36,7 @@ def parallel_executor
         def build_parallel_executor
           case parallelize_with
           when :processes
+            ActiveSupport.parallelize_test_databases = parallelize_databases
```

Comment:
It does not seem necessary for `ParallelizeExecutor` to know about this concern, couldn't the flag could be set directly from `parallelize`?

Reply:
I didn't do that because the hooks aren't available or used in the threaded parallelizer so setting it in `parallelize` would set it for both, but only be used by the process. I guess it's kind of odd it defaults to true then, but better than having it have it change when set and then do nothing.

Reply:
Changed my mind and moved it to `parallelize`.